### PR TITLE
Fix: machine2 regression test on buildkite

### PR DESCRIPTION
**Pull request by kwek20**
# Description

Our machine2 regression test fails one in ~~20 on buildkite.
After testing this locally for about 50x, we decided it is likely a race condition between buildkite and the node.
We increase the milestone wait time by 5 seconds and see if it ever fails again

## Type of change
- Bug fix (a non-breaking change which fixes an issue)

# How Has This Been Tested?

Ran the test a quazillion times,  fresh db every time

# Checklist:
- [X] My code follows the style guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: python-regression/tests/features/steps/response_handling_steps.py [hunk 1]
```diff
@@ -4,8 +4,10 @@
 from util.test_logic import api_test_logic as api_utils
 from util.test_logic import value_fetch_logic
 
-logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger(__name__)
+from util.response_logic import response_handling as response_handling
```
Comment (GalRogozinski) [inline python-regression/tests/features/steps/response_handling_steps.py]: Codacy Issue found: [redefinition of unused 'response_handling' from line 2](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=4946254)

Comment (GalRogozinski) [inline python-regression/tests/features/steps/response_handling_steps.py]: Codacy Issue found: [Reimport 'response_handling' (imported line 2)](https://app.codacy.com/manual/GalRogozinski/iri/pullRequest?prid=4946254)

**Comment by GalRogozinski** [comment]
@DyrellC 
Before I merge I am a bit confused... we said there is a fork in the original db that caused the error most likely no? So it is not the time?

@kwek20 
What are the 2 other older commits? I think they are good commits but just wondering, are there from other PRs that were not opened yet?

Just surprised to see them

**Comment by kwek20** [comment]
@GalRogozinski If you don't think its the time that breaks the commit, why would this PR be good?
I personally think this will fix the issue, my reasons are in the description.

**Comment by GalRogozinski** [comment]
I thought it was the time... then later @DyrellC told me the Db itself had a little fork

**Comment by kwek20** [comment]
@GalRogozinski Oh i dont know about that! Lets hear it from the man :D

**Comment by GalRogozinski** [comment]
After endless rerunning by @DyrellC and @kwek20 they have concluded it was k8s latency issue

**Comment by GalRogozinski** [comment]
Fixes #1607

**Comment by GalRogozinski** [comment]
@kwek20 
just notice now... can you change the base branch too release (no need to close and open a new PR).
If this indeed helps with the rare failure best to have it on the release branch...

Murphy's law dictates we will see the failure on the Release pipeline otherwise

**Comment by kwek20** [comment]
@GalRogozinski fixed!

**Pull request merged**